src/checkbox/src/Checkbox.tsx

```diff
@@ -153,6 +154,9 @@ export default defineComponent({
       props,
       mergedClsPrefixRef
     )
+    const labelId = computed(() => generateId('n-checbox-label'))
+    const ariaLabelledBy = computed(() => labelId.value)
```

Comment:
No need for reactivity, just create a const and expose it to template.

The value won't change in the entire component life cycle.